packages/devtools_app/lib/src/extensions/extension_service.dart

```diff
@@ -198,3 +299,62 @@ Future<Uri?> _connectedAppRoot() async {
   if (packageUriString == null) return null;
   return Uri.parse(packageUriString);
 }
+
+extension on DevToolsExtensionConfig {
```

Comment:
I don't want to add this directly to the `DevToolsExtensionConfig` class because it is specific to the DevTools extensions service, and doesn't need to be in the model class. It seemed cleaner to be able to call `extension.ignore()` and `extension.ignored` vs having to call some method `ignoreExtension(extension)` and `isExtensionIgnored(extension)`, but I could be convinced that these should be methods in `ExtensionService` if you think there would be a benefit to that. I'll admit I don't love that `_ignoredStaticExtensionsByHashCode` is a top level variable instead of being part of the `ExtensionService` class.

Reply:
I convinced myself with my comment. Updated to use methods and keep `_ignoredStaticExtensionsByHashCode` inside the class. That way, if there are multiple instances of ExtensionService, they will not share the same `_ignoredStaticExtensionsByHashCode` set.

Reply:
Ah, okay that makes sense. I didn't realize `DevToolsExtensionConfig` didn't hold any state. Moving `_ignoredStaticExtensionsByHashCode` into `ExtensionService` sounds like a good change.